lib/shared/bucketing-assembly-script/__tests__/bucketing/bucketing.test.ts

```diff
@@ -163,6 +163,11 @@
             'featureVariationMap': {
                 '614ef6aa473928459060721a': '615357cf7e9ebdca58446ed0'
             },
+            'variableFeatureVariationMap': {
```

Comment:
Let's just call this the `variableVariationMap`, since that's what it really is.